mod-orgs/schemas/account.json

```diff
@@ -13,35 +13,45 @@
       "description": "The unique name of this account",
```

Comment:
Could you please fix doubled `description` for `id` property above?

Reply:
Do we even need an id after we removed the API for accounts?